pages/user guides/code_research.md

```diff
@@ -0,0 +1,51 @@
+---
+toc: false
+title: Tidal Code Research Program
+keywords: code, analyze, source code
+last_updated: Novemvber, 2023
+summary: "Participate in the Code Research Program and gain even more insights from your code"
+sidebar: main_sidebar
+folder: userguides
+permalink: code-research.html
+---
+
+## Benefits of Participating
+
+- You will get early access to new findings and insights for your entire application portfolio, not only the ones that are a part of the program, at no additional cost.
+- Providing the data will allow us to do extensive research in the analysis and recommendations for migrating applications.
+- You can opt out and remove your code at any time, let us know if you want to leave and we will immediately remove any code you provided.
+
+## What we do and not do with the Code
+- We will use the code to identify new and confirm and validate configurations and implementation details that should be considered when migrating or transforming an application.
```

Comment:
I think this might be worthwhile breaking up into multiple statements. 

Identify new what? New configurations?
If so can we remove the confirm and validate?
>... identify new configurations and implementation details that should be considered when migrating or transforming an application.

or if not
> ...identify, confirm, and validate new configurations and implementation details that should be considered when migrating an application.

Reply:
Good suggestion thanks